fortran/flake.nix

```diff
@@ -0,0 +1,24 @@
+{
+  description = "A Nix-flake-based Fortran development environment";
+
+  inputs.nixpkgs.url = "https://flakehub.com/f/NixOS/nixpkgs/0.1.*.tar.gz";
+
+  outputs = { self, nixpkgs }:
+    let
+      supportedSystems = [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
+      forEachSupportedSystem = f: nixpkgs.lib.genAttrs supportedSystems (system: f {
+        pkgs = import nixpkgs { inherit system; };
+      });
+    in
+    {
+      devShells = forEachSupportedSystem ({ pkgs }: {
+        default = pkgs.mkShell {
+          packages = with pkgs; [ fortran-fpm ];
+        };
+      });
+      shellHook = ''
```

Comment:
Aliases don't work in shell hooks, I'm afraid. I'd recommend not having a shell hook here and enabling others to add one if they wish.